test/helpers/clusterlogging.go

```diff
@@ -51,6 +51,11 @@ func NewClusterLogging(componentTypes ...LogComponentType) *cl.ClusterLogging {
 					NodeCount:        1,
 					RedundancyPolicy: elasticsearch.ZeroRedundancy,
 				},
+				RetentionPolicy: &cl.RetentionPoliciesSpec{
```

Comment:
looking at the code it looks like we just check if the fields are `nil` so we could possibly just have:
```
RetentionPolicy: &cl.RetentionPoliciesSpec{
  App: &cl.RetentionPolicySpec{},
  Infra: &cl.RetentionPolicySpec{},
  Audit: &cl.RetentionPolicySpec{},
}
```

Reply:
https://github.com/openshift/cluster-logging-operator/pull/426

Reply:
maybe not.. `"Error occured while getting hot phase age for App log source. err: age pattern mismatch"`